### PR TITLE
[let me leave this as wip until i make it use vv_edit_var tmrw]More SDQL update safeguards

### PR DESCRIPTION
**Pull request by silicons**
I AM NOT INSECURE ABOUT MY BUTTON PRESSING.

### CODE REVIEW

File: code/controllers/configuration.dm [hunk 1]
```diff
@@ -13,6 +13,9 @@
 /datum/protected_configuration/vv_get_var(var_name)
 	return debug_variable(var_name, "SECRET", 0, src)
 
+/datum/protected_configuratoin/SDQL_update()
```
Comment (JJRcop) [inline code/controllers/configuration.dm]: toin

Comment (JJRcop) [inline code/controllers/configuration.dm]: So.. doc, what's the deal with my leg muscle? Why isn't it moving?

It was toin.

**Comment by optimumtact** [comment]
you should make this use view var like mso said

**Comment by kevinz000** [comment]
remind me to do that later i'm sleepy as fuck.

**Comment by kevinz000** [comment]
@optimumtact wait what if I want SDQL to be more restrictive than VV in terms of what vars they can edit, because honestly no one needs to mass edit the contents of atoms?

**Pull request closed without merging**